taca/storage/storage.py

```diff
@@ -239,21 +241,22 @@ def _send_to_swestore(file_path, dest, remove=True):
         if os.stat(run).st_mtime < time.time() - (86400 * days):
             _send_to_swestore(run, CONFIG.get('storage').get('irods').get('irodsHome'))
         else:
-            logger.info("Run {} is not {} days old yet. Not archiving"
-                        .format(run, str(days)))
+            logger.info("Run {} is not {} days old yet. Not archiving".format(run, str(days)))
     else:
         rta_file = os.path.join(run, 'RTAComplete.txt')
-        if os.stat(rta_file).st_mtime < time.time() - (86400 * days):
+        if not os.path.exists(rta_file) and not force:
+            logger.warn(("Run {} doesn't seem to be completed and --force option was "
+                      "not enabled, not archiving the run".format(run)))
```

Comment:
Yes, when the first `if` is `True`, the second `if` will not be `True` :). But my concern is when a `RTAComplete.txt` file is not present, it will go inside the first `if` and than again while trying to the second `if` it will fail with `OSError` since the file is not present and it cant get `os.stat`. Again since we are using `Pooling` this will not make the script fail, but that process will silently fail.

So may maybe changing the second `if` to `elif` might not change the functionality and still remove the `stat` file in the end ?? :eyes: